components/SignUpForm.tsx

```diff
@@ -0,0 +1,118 @@
+"use client";
+
+import Image from "next/image";
+import Link from "next/link";
+import { useRef, useState } from "react";
+
+// Logo Import
+import logo from "../public/Cacart_logo.png";
+import googlelogo from "../public/google_logo.png";
+
+type FormInputs = {
+  email: string;
+  password: string;
+};
+
+export default function SignUpForm() {
+  // Set userData
```

Comment:
I think these types of comments where the function names are simply repeated are not necessary. We don't need to be excessive with the comments as the code should explain what it does. That being said, having comments explaining specific business related logics or domain related logics would be suggested.

Reply:
In the latest commit, I've rearranged functions and constants for better readability and added comments where deemed necessary. Please review it.